pkg/processor/trigger/kickstart/types.go

```diff
@@ -37,7 +37,11 @@ func NewConfiguration(id string,
 	newConfiguration := Configuration{}
 
 	// create base
-	newConfiguration.Configuration = *trigger.NewConfiguration(id, triggerConfiguration, runtimeConfiguration)
+	baseConfiguration, err := trigger.NewConfiguration(id, triggerConfiguration, runtimeConfiguration)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
wrap errors, or else your traceback would not point to this error

Reply:
done